rustfmt-core/rustfmt-lib/src/formatting/comment.rs

```diff
@@ -867,6 +868,12 @@ fn has_url(s: &str) -> bool {
     s.contains("https://") || s.contains("http://") || s.contains("ftp://") || s.contains("file://")
 }
 
+/// Returns true if the given string may be part of a Markdown talble.
+fn is_table_item(s: &str) -> bool {
+    // This function may return false positive, but should get its job done in most cases.
+    s.trim_start().starts_with('|')
```

Comment:
Agreed there's probably not a ton of non-table instances with lines that start with pipes. However, could we reduce false positives by also checking that there's a minimum of two pipes or would that result in a greater quantity of missed tables (presumably single column tables without the trailing pipe)?

Reply:
That sounds reasonable, I don’t think single-column tables stretching past 80 columns are very common.